src/utils/decryptAES.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2024 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { decodeBase64 } from "../base64.ts";
+import { deriveKeys } from "../rust-crypto/deriveKeys.ts";
+import { SecretEncryptedPayload } from "./@types/SecretEncryptedPayload.ts";
+
+/**
+ * Decrypt an AES-encrypted string.
+ *
+ * @param data - the encrypted data, returned by {@link encryptAES}.
```

Comment:
These links don't work. Try something like:

```suggestion
@param data - the encrypted data, returned by {@link utils/encryptAES.default | encryptAES}.
```

(Assuming you're using default exports.)